### PR TITLE
Add layout of the login with email and password

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Add layout of the login with email and password.

#### How should this be manually tested?
https://waza--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/EmailAndPasswordVerification.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    console.log('Email:', email)
+    console.log('password:', password)
+  }
+
+  render() {
+    const { goBack, send, intl, onStateChange, previous, email, password, titleLabel } = this.props
+    const { isLoading } = this.state
+
+    return (
+      <div className="vtex-login_
```
Comment (brunojdo) [inline react/components/EmailAndPasswordVerification.js]: typo

File: react/components/EmailAndPasswordVerification.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  }
+
+  componentWillUnmount() {
+    this.setState({ isLoading: false })
+  }
+
+  handleOnSubmit = () => {
+    const { email, password } = this.props
+
+    console.log('Email:', email)
```
Comment (brunojdo) [inline react/components/EmailAndPasswordVerification.js]: remove

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Layout of the traditional login with email and password verification.
```
Comment (andreldsa) [inline CHANGELOG.md]: Login with email and password.

File: react/Login.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { graphql } from 'react-apollo'
 import { Button } from 'vtex.styleguide'
 import LoginOptions from './components/LoginOptions'
 import EmailVerification from './components/EmailVerification'
+import EmailAndPasswordVerification from './components/EmailAndPasswordVerification'
```
Comment (andreldsa) [inline react/Login.js]: EmailAndPassword*

File: react/Login.js [hunk 5]
```diff
@@ -20,7 +21,7 @@ const STEPS = [
       <LoginOptions
         page="login-options"
         titleLabel="login-options.title"
-        options={['login-options.email-verification']}
+        options={['login-options.email-verification', 'login-options.email-and-password-verification']}
```
Comment (andreldsa) [inline react/Login.js]: remove verification from all references.

File: react/components/EmailAndPassword.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    this.props.onStateChange({ password: event.target.value })
+  }
+
+  componentWillUnmount() {
+    this.setState({ isLoading: false })
+  }
+
+  handleOnSubmit = () => {
+    const { email, password } = this.props
+
```
Comment (andreldsa) [inline react/components/EmailAndPassword.js]: Add event.preventDefault() to avoid page refresh

**Pull request merged**